.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+## Description
```

Comment:
I appreciate your effort of setting up a PR template; however, I would remove it.

Currently, most of our PR are either done by Polar team or by contributors we already gave guidance to on Discord and/or GitHub issues. So, I don't think we need it right now :)